tools/docker-compose/ansible/requirements.yml

```
@@ -1,6 +1,7 @@
 ---
 collections:
-  - source: ./awx_collection
+  - name : awx_collection
```

I don't think this is the _right_ name. I expect this is usually running with it packaged as `awx.awx`.

---

It's
```suggestion
  - name: awx.awx
```

But really, using both source and name isn't typically necessary: https://docs.ansible.com/ansible/latest/collections_guide/collections_installing.html#installing-a-collection-from-source-files.